src/key.cpp

```diff
@@ -74,23 +73,28 @@ CPubKey CKey::GetPubKey() const {
     return result;
 }
 
+extern "C"
```

Comment:
@gmaxwell just suggested adding functionality in libsecp256k1 for its rfc5979 implementation to accept extra entropy, which could be used for the 'test_case' function too, in a much cleaner way.

The change here is 100% compatible with the existing code though, so unit tests relying on it don't break.